tests/ui/codegen/opaque-cast-monormophize.rs

```diff
@@ -0,0 +1,13 @@
+// build-pass
```

Comment:
this one as well, while this is a codegen test, it probably makes more sense together with the above test